ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -343,9 +343,13 @@ public String processImport(ImportDeclarationSnippet importSnippet) throws Invok
 
         Set<GlobalVariable> foundVariables = new HashSet<>();
         for (Symbol symbol : symbols) {
+            if (symbol.getName().isEmpty()) {
```

Comment:
Does this mean that we do not process service symbols or the service symbol has a generated name?

Reply:
Service symbol isn't added in this PR. Depending on the path we take for service symbols, I guess we'll have to revisit this.